recipes/cantools/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "cantools" %}
+{% set version = "20.6.0" %}
+{% set sha256 = "1cc876ec1c8de17bafc6f631292d9d71902da06aae6a394ec6fc3f4f5518389f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+  entry_points:
+    - cantools=cantools.__init__:_main
+
+requirements:
+  build:
+    - python
+    - pip
+    - bitstruct>=3.7.0
+    - pyparsing>=2.0.3
+    - python-can>=2.1.0
+  run:
+    - python
+    - bitstruct>=3.7.0
+    - pyparsing>=2.0.3
+    - python-can>=2.1.0
+
+test:
+  imports:
+    - cantools
+
+about:
+  home: https://github.com/eerimoq/cantools
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'CAN BUS tools.'
+  description: |
+    CAN BUS tools
+    - `DBC`_, `KCD`_ and SYM file parsing.
+    - CAN message encoding and decoding.
+    - Simple and extended signal multiplexing.
+    - ``candump`` output decoder.
+    - Node `tester`_.
+  doc_url: http://cantools.readthedocs.io/en/latest/
+  dev_url: https://github.com/eerimoq/cantools
+
+extra:
+  recipe-maintainers:
+    - eerimoq
```

Comment:
@eerimoq are you ok in being a maintainer of this recipe?

Reply:
I would prefer not to as I don't have time, nor intrest, in being so.

Reply:
@danielballan , please remove @eerimoq from this recipe